engine/runner/src/main/scala/org/enso/runner/ContextFactory.scala

```diff
@@ -95,11 +105,26 @@ class ContextFactory {
           new DebuggerSessionManagerEndpoint(repl, peer)
         } else null
       }
-      .option(
+
+    val logHandler = JulHandler.get()
+    val logLevels  = LoggerSetup.get().getConfig.getLoggers
+    if (!logLevels.hasEnsoLoggers(LanguageInfo.ID)) {
+      builder.option(
         RuntimeOptions.LOG_LEVEL,
         logLevelName
       )
-      .logHandler(JulHandler.get())
+    } else {
+      builder.option(
+        RuntimeOptions.LOG_LEVEL,
+        java.util.logging.Level.ALL.getName
+      )
```

Comment:
I believe we have to enumerate all the `enso....Logger.level` properties and apply them one by one with `log.enso...` prefix.